packages/amplify_authenticator/lib/src/state/auth_viewmodel.dart

```diff
@@ -337,15 +337,10 @@ class AuthViewModel extends ChangeNotifier {
   }
 
   Future<void> resendSignUpCode() async {
-    if (!_formKey.currentState!.validate()) {
-      return;
-    }
-    setBusy(true);
```

Comment:
Similarly to above, `setBusy` sets the form to busy. The form's primary button goes to a loading state. Resending the code doesn't submit the form though. This makes it look like the form was submitted, but it wasn't.